data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	The monitor also allows you to switch to a camera view of the entire track. It is a long, mostly flat stretch of land, leading to a rocky cliff wall on the far side that is cut to form a perfect right angle with the ground. Just beyond the top of the cliff is a river bank with many logs laid on the ground. The river itself is not terribly wide, but the current is so strong that the various Heliarch safety boats already waiting in the waters need to constantly adjust their position in order to avoid hitting the many rock outcroppings protruding from the water. Most significantly, though, as the river approaches the tree line, it drops off in a dangerous looking waterfall to resume its course on the lower ground.`
+			`	Pairri draws your attention back to the starting line, where Heliarch judges have the twenty teams line up. The Saryds, who seem to be working themselves up by hitting their hoofs repeatedly on the ground wait otherwise patiently. Meanwhile, their Kimek and Arach teammates pack and secure large bags and lengths of fabric in what appears to be a more conventional chariot. When the packing is complete and their teammates have boarded, the Saryds lift the yoke and strap themselves in before returning to their hoof stamping. As each team readies themselves, the Kimek and Arachi in the chariots pound on the side in time to the stamping and the teams begin to synch up, creating a uniform rhythm, like a giant heart beat.`
+			`	"In many ways we differ, when it comes to physical form," Pairri says. "So in the cooperative match, when come together as one we do, different tasks, each member of the team is assigned. For the sake of fairness, it is so."`
+			choice
+				`	"What are they carrying in those bags?"`
+				`	"So is pulling those chariots the task of the Saryds?"`
+			`	A loud airhorn interrupts Pairri as she is about to answer, and the Saryds start galloping toward the wall. The Kimek and Arachi in each chariot do their best to stay on, as their Saryd teammates seem to generally press on with little regard for the chariot's stability; instead they are focused solely on reaching the wall as fast as possible.`
+			`	It's a very tight race, but the Saryds begin slowing once they approach the wall, both out of exhaustion and to prevent a crash. As the Saryds arrive at the base of the cliff, they immediately lay down to rest while the Arach on their team begins to unstrap the chariot. The Kimek hurriedly begins climbing the wall with the bags they had packed earlier.`
+			`	Before long, the Kimek have all reached the top of the cliff and begin opening their bags to grab a multitude of paddles, some kind of pole which looks vaguely like a combination shovel and a pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must. My role this would have been. As skilled in anticipating the actions of my friends I hope my former opponent is."`
```

Comment:
```suggestion
			`	Before long, the Kimek have all reached the top of the cliff and begin opening their bags to grab a multitude of paddles, some kind of pole which looks vaguely like a combination of a shovel and pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must. My role this would have been. As skilled in anticipating the actions of my friends, I hope my former opponent is."`
```